### PR TITLE
Add workspace host load counter

### PR DESCRIPTION
**Pull request by tdy**

### CODE REVIEW

File: workspace_host/interface.sql [hunk 1]
```diff
@@ -17,3 +17,23 @@ INSERT INTO workspace_hosts
...
+
+-- BLOCK decrement_load_count
+UPDATE workspace_hosts as wh
+SET
+    load_count = load_count - 1
+FROM
+    workspaces as w
+WHERE
+    w.id = $workspace_id
+    AND w.workspace_host_id = wh.id;
```
Comment (tdy) [inline workspace_host/interface.sql:39]: somehow this feels wrong (any way to refactor this?)

File: database/tables/workspace_hosts.pg [hunk 2]
```diff
@@ -2,6 +2,7 @@ columns
     hostname: text
     id: bigint not null default nextval('workspace_hosts_id_seq'::regclass)
     instance_id: text
+    load_count: smallint default 0
```
Comment (mwest1066) [inline database/tables/workspace_hosts.pg:5]: Just use `integer`

Comment (tdy) [inline database/tables/workspace_hosts.pg:5]:
```suggestion
    load_count: integer default 0
```

File: migrations/189_workspace_hosts__load_count__add.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE workspace_hosts ADD COLUMN IF NOT EXISTS load_count smallint default 0;
```
Comment (tdy) [inline migrations/189_workspace_hosts__load_count__add.sql:1]:
```suggestion
ALTER TABLE workspace_hosts ADD COLUMN IF NOT EXISTS load_count integer default 0;
```

**Pull request merged**